addons/web/static/tests/webclient/actions/embedded_action.test.js

```diff
@@ -277,17 +287,17 @@ test("breadcrumbs are updated when clicking on embeddeds", async () => {
     ).click();
     expect(".o_control_panel .breadcrumb-item").toHaveCount(0);
     expect(".o_control_panel .o_breadcrumb .active").toHaveText("Partners Action 1");
-    expect(browser.location.href).toBe("https://www.hoot.test/odoo/action-1");
+    expect(browser.location.href).toBe("https://www.hoot.test/odoo/1/action-1");
```

Comment:
This is a shame, and relates to my comment above about resId VS currentActiveId
here, the activeId is 1, but doesn't mean anything because there is no previous action.
This url is not elegant and less human readable than before.